extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/SmallRyeReactiveMessagingProcessor.java

```diff
@@ -211,8 +262,282 @@ public void build(SmallRyeReactiveMessagingRecorder recorder, BeanContainerBuild
                 recorder.configureEmitter(beanContainer.getValue(), it.getName(), null, 0, defaultBufferSize);
             }
         }
+    }
+
+    private QuarkusMediatorConfiguration createMediatorConfiguration(MethodInfo methodInfo, BeanInfo bean,
```

Comment:
I would move this long method into its own class.

Reply:
Done